Validation/fcl/potSimMT.fcl

```diff
@@ -7,6 +7,7 @@
 # set in the non MT file included above (which includes prolog files)
 
 physics.producers.g4run.module_type : "Mu2eG4MT"
+physics.producers.g4run.initialSeed  : 10
```

Comment:
Another thing on  the list if we can use the SeedService to seed the master engine.

Reply:
I have removed the initialSeed FHICL parameter. We are now using the Seed Service to provide the seed to the random number generator used by the Mu2eG4 master run manager.